src/app/core/standard-guidelines-module/standard-guidelines.service.ts

```diff
@@ -15,7 +15,7 @@ export class GuidelineService {
   constructor(public http: HttpClient) { }
 
   getGuidelines(
-    filter?
+    filter?: any
```

Comment:
Are we able to add a more specific type other then any?

Reply:
There is probably a type for this over in standard guidelines: https://github.com/Cyber4All/standard-guidelines-service